src/sidebar/services/test/frame-sync-test.js

```diff
@@ -591,6 +604,30 @@ describe('FrameSyncService', () => {
       await connectGuest();
       assert.calledWith(channel.call, 'setHighlightsVisible', false);
     });
+
+    [true, false].forEach(haveContentInfo => {
```

Comment:
Deciphering the context of `true` and `false` here took me a few glances, so I wonder if being more explicit here is possible. `haveContentInfo` is a helpful param name, but it's still "invented" for this test. Perhaps even a tiny tweak to bring the param name a little closer to the test description, e.g. `contentInfoAvailable`? Minor, but there you have it...

Reply:
OK, I'll rename the variable as suggested.